gomp panic: invalid memory address

Just found this project, looks interesting. Installed it from aur, but ran into this error. 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x563de28a0cf3]

goroutine 6 [running]:
gitlab.com/diamondburned/ueberzug-go.NewImage({0x0, 0x0}, 0x0?, 0x0?)
	gitlab.com/diamondburned/ueberzug-go@v0.0.0-20190521043425-7c15a5f63b06/ueberzug.go:25 +0x53
github.com/aditya-K2/gomp/render.OpenImage({0xc000165946, 0x3a}, 0x0?)
	github.com/aditya-K2/gomp/render/render.go:52 +0x110
created by github.com/aditya-K2/gomp/render.(*Renderer).Start
	github.com/aditya-K2/gomp/render/render.go:70 +0x8a
```
Cloned the git repo, built it from there, same error. Then I checked out to an earlier commit. Same error. Seems it might be something on my side. 
Here is my config.yml
```
MPD_PORT : "6600"
MUSIC_DIRECTORY : "~/Music"
DEFAULT_IMAGE_PATH : "~/Pictures/defaultlain.jpg"
SEEK_OFFSET : 5
CACHE_DIR : "~/.config/gomp/cache"
#GET_COVER_ART_FROM_LAST_FM : "TRUE"
#LASTFM_API_KEY : "4e316e10a94a2eef30b605fc3db04cf1"
#LASTFM_API_SECRET : "d95f07d336316312a81e34680c6d5bb2"
```
Please advise. Also, sorry for making an empty issue and then editing it later. Was about to type this up when I misclicked and then something came up.

--- a/config/config.go
+++ b/config/config.go
@@ -29,15 +29,23 @@ var (
 )
 
 func ReadConfig() {
+
 	for k, v := range defaults {
 		viper.SetDefault(k, v)
 	}
+
 	viper.SetConfigName("config")
 	viper.AddConfigPath(HOME_DIR + "/.config/gomp")
+
 	err := viper.ReadInConfig()
 	if err != nil {
 		fmt.Println("Could Not Read Config file.")
 	}
+
+	// Expanding ~ to the User's Home Directory
+	viper.Set("MUSIC_DIRECTORY", utils.ExpandHomeDir(viper.GetString("MUSIC_DIRECTORY")))
+	viper.Set("DEFAULT_IMAGE_PATH", utils.ExpandHomeDir(viper.GetString("DEFAULT_IMAGE_PATH")))
+	viper.Set("CACHE_DIR", utils.ExpandHomeDir(viper.GetString("CACHE_DIR")))
 }
 
 func GenerateKeyMap(funcMap map[string]func()) {

--- a/utils/utils.go
+++ b/utils/utils.go
@@ -2,6 +2,8 @@ package utils
 
 import (
 	"io/ioutil"
+	"os"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"syscall"
@@ -114,6 +116,17 @@ func CheckDirectoryFmt(path string) string {
 		return path
 	} else {
 		return path + "/"
+	}
+}
+
+func ExpandHomeDir(path string) string {
+	HOME_DIR, _ := os.UserHomeDir()
+	if strings.HasPrefix(path, "~/") {
+		return filepath.Join(HOME_DIR, path[1:])
+	} else if path == "~" {
+		return HOME_DIR
+	} else {
+		return path
 	}
 }
 

